src/Umbraco.Cms.Integrations.Crm.Hubspot/App_Plugins/UmbracoCms.Integrations/Crm/Hubspot/Render/HubspotForm.cshtml

```diff
@@ -1,31 +1,14 @@
-﻿@using System.Configuration
-@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Cms.Integrations.Crm.Hubspot.Models.ViewModels.HubspotFormViewModel>
+﻿@inherits Umbraco.Web.Mvc.UmbracoViewPage<Umbraco.Cms.Integrations.Crm.Hubspot.Models.ViewModels.HubspotFormViewModel>
 
 @{
-    var region = ConfigurationManager.AppSettings["Umbraco.Cms.Integrations.Crm.Hubspot.Region"];
-
-    var hbsptScriptPath = string.IsNullOrEmpty(region) ? "//js.hsforms.net/forms/shell.js" : "//js-eu1.hsforms.net/forms/shell.js";
+    var hbsptScriptPath = string.IsNullOrEmpty(Model.Region) ? "//js.hsforms.net/forms/shell.js" : "//js-eu1.hsforms.net/forms/shell.js";
```

Comment:
Should `eu1` here in the script's host name actually come from the value of `Model.Region`?  In case there is more than one region?